Octo-Bot/healthcare-registration-bot/test/test.py

```diff
@@ -0,0 +1,86 @@
+import io
+import sys
+import unittest
+import warnings
+
+from BotActions import (adminLogin, assignTherapist, getDriver,
+                        getDriverStatus, registerPatientAccount)
+from FileParser import (getAdminCredentials, getPatientCredentials,
+                        getTherapistCredentials)
+from main import main
+
+
+class Testing(unittest.TestCase):
+    '''
+    Unit tests to test that csv files can be read and driver operates as expected
+    If any tests fail, please check for regression
+
+    Expected output:
+        ......
+        ----------------------------------------------------------------------
+        Ran 7 tests in X.XXXs
+        OK        
+    ''' 
+    def setUp(self):
+        warnings.simplefilter('ignore',category = DeprecationWarning)
+        warnings.simplefilter('ignore', category = ResourceWarning)
+        text_trap = io.StringIO()
+        sys.stdout = text_trap
+
+    def testGetAdminCredentials(self):
+        adminCredentials = getAdminCredentials()
+        adminCredentialsData = ["S0000001A", "DrasticP@ssw0rdPlayer"]
+        self.assertEqual(adminCredentials, adminCredentialsData)
+    
+    def testGetTherapistCredentials(self):
+        therapistCredentials = getTherapistCredentials()
+        therapistCredentialsData = ["F0058132Q", "LightEnterP@ssw0rd", "Z9989"]
+        self.assertEqual(therapistCredentials, therapistCredentialsData)
+    
+    def testGetPatientCredentials(self):
+        patientCredentials = getPatientCredentials()
+        patientCredentialsData = ["S1757946A", "1995-12-12", "John", "Tan", "Block 5002 Ang Mo Kio Avenue 5 04-05 TECHplace II Singapore 569871  Singapore", "john@email.com", "569871", "12345678", "veryEasyP@ssw0rd", "Weight", "20"]
```

Comment:
Can we hide this password? Ex. use our own defined patient.csv

Reply:
Resolved!